ClientApp/src/components/landingPage/Macro.tsx

```diff
@@ -9,10 +9,22 @@ import logo from "./../../styling/img/logo_background.svg";
 import family from "./../../styling/img/familyTop.svg";
 import snowDown from "./../../styling/img/snow_down.svg";
 import Contact from "./Contact";
+import {ContactsData} from "./Contact";
 import NavBar from "../../common/components/NavBar";
 import Footer from "../../common/components/Footer";
+import { useState } from "react";
+import { useEffect } from "react";
+import ApiService from "../../common/functions/apiServiceClass";
 
 const Home = () => {
+
+  const [xContact, setContactCards] = useState<ContactsData[]>([]);
```

Comment:
xContact and setContactCards is inconsistent and hard to understand. 

https://reactjs.org/docs/hooks-state.html
states should always be named: [myState, setMyState] 

I suggest naming the state: contacts